### PR TITLE
Add an email address to the register page and redirect based on it

### PR DESCRIPTION
**Pull request by TiannaW24**
redirect to ADFS if .dot.gov email
redirect to login.gov for any other email address
_minimal styling and character masking_

### CODE REVIEW

File: webportal/webapp/src/app/main/register/register.component.ts [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 export class RegisterComponent implements OnInit {
+  email: string;
 
   constructor(private cognitoService: CognitoService) { }
   ngOnInit() {
   }
   userLogin() {
-      this.cognitoService.login(false);
+    if (this.email.endsWith('dot.gov')) {
+      this.cognitoService.login(false); // Route to ADFS login
```
Comment (aaronFranssell) [inline webportal/webapp/src/app/main/register/register.component.ts]: Is it possible to make this a named parameter? e.g. `loggedIn: false`

Comment (TiannaW24) [reply webportal/webapp/src/app/main/register/register.component.ts]: stackoverflow says no: https://stackoverflow.com/questions/14931824/typescript-named-parameters-like-used-in-angularjs

Comment (TiannaW24) [reply webportal/webapp/src/app/main/register/register.component.ts]: theres actually a little bit of a hubbub about it: https://github.com/Microsoft/TypeScript/issues/467

Comment (TiannaW24) [reply webportal/webapp/src/app/main/register/register.component.ts]: but the short answer is no

**Pull request merged**